CodeBeam.MudExtensions/Components/CsvMapper/MudCsvMapper.razor

```diff
@@ -0,0 +1,107 @@
+﻿@namespace MudExtensions
+@inherits MudComponentBase
+
+@using Microsoft.AspNetCore.Components
+@using System.ComponentModel.DataAnnotations
+@using Microsoft.AspNetCore.Components.Forms
+
+<div class="@Class" style="@Style">
+
+<style>
+	.mud-drop-item{
+	    flex-shrink: 0 !important;
+		display:inline-block !important;
+		margin: 0 4px;
+	}
+	</style>
+
+	<style>
+		.mud-drop-item {
+			flex-shrink: 0 !important;
+			display: inline-block !important;
+			margin: 0 4px;
+		}
+	</style>
+
+	<MudStack Style="width: 100%">
+		<MudFileUpload T="IBrowserFile" Accept=".csv" OnFilesChanged="OnInputFileChanged" Hidden="false" Class="flex-1 d-flex justify-center align-content-center my-2"
+					   InputClass="absolute mud-width-full mud-height-full d-flex justify-center align-content-center overflow-hidden z-20" InputStyle="opacity:0"
+					   @ondragenter="@SetDragClass" @ondragleave="@ClearDragClass" @ondragend="@ClearDragClass">
+			<ButtonTemplate>
+				<MudPaper Outlined="true" Class="@("d-flex flex-column justify-center align-content-center mud-background-gray p-2 " + DragClass)">
+					<MudPaper Elevation="0" class="mud-transparent d-flex justify-center align-content-center">
+						<svg class="mud-icon-root" viewBox="0 0 24 24" height="75" width="75" style=" height:75px; width:175px;" aria-hidden="true">
+							<path d="M0 0h24v24H0V0z" fill="none"></path>
+							<path d="M19.35 10.04C18.67 6.59 15.64 4 12 4 9.11 4 6.6 5.64 5.35 8.04 2.34 8.36 0 10.91 0 14c0 3.31 2.69 6 6 6h13c2.76 0 5-2.24 5-5 0-2.64-2.05-4.78-4.65-4.96zM19 18H6c-2.21 0-4-1.79-4-4 0-2.05 1.53-3.76 3.56-3.97l1.07-.11.5-.95C8.08 7.14 9.94 6 12 6c2.62 0 4.88 1.86 5.39 4.43l.3 1.5 1.53.11c1.56.1 2.78 1.41 2.78 2.96 0 1.65-1.35 3-3 3zm-5.55-8h-2.9v3H8l4 4 4-4h-2.55z"></path>
+						</svg>
+					</MudPaper>
+					<MudPaper Elevation="0" class="d-flex justify-center align-content-center mud-transparent">
+						<h3>
+							<MudFab HtmlTag="Button"
+									Color="Color.Secondary"
+									Label="Choose a .csv file"
+									Icon="@Icons.Outlined.CloudDownload" Size="Size.Large"
+									for="@context" /> or Drag and Drop a CSV file here.
```

Comment:
The texts are hard coded. Does a user need localization? If true we may prepare a localized text parameter like we did in MudStepper.